backend/src/v5/routes/teamspaces/projects/containers/containers.js

```diff
@@ -43,6 +43,24 @@ const getContainerStats = async (req, res) => {
 	}).catch((err) => respond(req, res, err));
 };
 
+const getRevisions = async (req, res) => {
+	const { teamspace, container } = req.params;
+	const showVoid = req.query.showVoid === 'true';
+
+	Containers.getRevisions(teamspace, container, showVoid).then((revisions) => {
+		respond(req, res, templates.ok, revisions);
```

Comment:
```suggestion
		respond(req, res, templates.ok, { revisions } );
```

It's a better practice to avoid returning arrays as the root level (also makes life a lot easier in the C# world where you need to serialise the json - as arrays aren't recognised as a top level thing...)

So for v5 we want to avoid returning arrays, wrap them in a json